site/docs/v1/tech/admin-guide/upgrade.adoc

```diff
@@ -4,44 +4,300 @@ title: Upgrade FusionAuth
+* You have concerns about compatibility with customizations or third-party integrations.
+
+=== Being Aware of Upgrade Options
+
+You can stay informed about new FusionAuth releases and updates by:
+
+* Monitoring the link:/docs/v1/tech/release-notes[release notes].
+* Joining the https://fusionauth.io/community[FusionAuth community] forum and/or slack.
+* Following FusionAuth on social media platforms, including https://twitter.com/FusionAuth[Twitter] and https://www.linkedin.com/company/fusionauth[LinkedIn].
+* Monitoring the https://github.com/FusionAuth/fusionauth-issues/issues[FusionAuth Issues GitHub repository].
+
+== How to Think About Upgrades
+
+Before upgrading FusionAuth, there are a number of things to consider. Upgrading FusionAuth is similar to upgrading a library your application depends on. Go through each of the points below to see if any of them apply to your situation.
+
+=== Update Contents and Notes
+
+Read the link:/docs/v1/tech/release-notes[release notes]. Release notes help you understand how an update affects you. They include details on new features, improvements, bug fixes, and potential breaking changes.
+
+=== Testing
+
+Before starting the upgrade process on production, test the upgrade in a staging or development environment to identify potential issues. This is particularly important on more complex upgrades involving multiple version jumps or if you have made customizations to FusionAuth.
+
+=== Downtime and Database Migrations
+
+Plan for any downtime or maintenance windows required for the upgrade. If changes to the database schema are required, this could require more steps and downtime that affects end users. Such changes are always noted in the release notes. (Read the release notes carefully.) A pure application upgrade, on the other hand, may not require any downtime from an end-user perspective.
+
+FusionAuth can perform database migrations automatically and silently using a process called link:/docs/v1/tech/guides/silent-mode[Silent Mode]. 
+
+If you want to run the migrations interactively, you can run any needed scripts out of band (meaning directly on the database yourself). The general process would be to: 
+
+- Remove all server nodes from a load balancer/proxy.
+- Run the database migration scripts.
+- Add all server nodes back to the load balancer/proxy.
 
 [NOTE]
 ====
-*Readme*
+You should always back up your database before using Maintenance Mode or performing any database migrations by any method.
+====
 
-To upgrade to the latest version of FusionAuth using the FastPath installation go link:/docs/v1/tech/installation-guide/fast-path[here].
+The timing of the database upgrade is also important to note. When an upgraded node is started and connects to the database, it will lock the database immediately if configured for Silent Mode and the database requires upgrading. This means that the database will be unavailable to other nodes in the cluster until the upgrade is complete.
```

Comment:
This is the first mention of Silent Mode and might be confusing. Can you talk about modes before this so folks don't get confused? The two modes that need to be covered are silent mode and runtime mode.